### ISSUE TITLE
How upload image

### ISSUE THREAD

**Issue by dannying** [Weakky/ra-data-prisma#27]
How can I upload images ? I added the "picture" field. when I try to select an image and click on the save, I get the error Cannot read property 'name' of null.

```
 <ImageInput source="images" label="Related pictures" accept="image/*">
        <ImageField source="create.src" />
 </ImageInput>
```

my mutation arguments: 
data: {name:"string", image: {create: src:"string"}}

**Comment by abdullah-dev11** [comment]
@dannying did you find any solution?

**Comment by dannying** [comment]
@abdullah-almesbahi No:( I decided to create admin framework for graphql, it quickly and right solution in my opinion. You can try to use ra-data-graphql adapter and graphql yoga, but I couldn't connect them. Maybe you be able
